mod-kitpvp/src/main/java/gg/solarmc/loader/kitpvp/BountyCurrency.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * DataLoader
+ * Copyright © 2021 SolarMC Developers
+ *
+ * DataLoader is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * DataLoader is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with DataLoader. If not, see <https://www.gnu.org/licenses/>
+ * and navigate to version 3 of the GNU Affero General Public License.
+ */
+
+package gg.solarmc.loader.kitpvp;
+
+import java.math.BigDecimal;
+
+/**
+ * The medium of exchange for bounties. <br>
+ * <br>
+ * Strictly speaking, the names of the enum entries are purely informative
+ * and do not impose requirements on the API user.
+ *
+ */
+public enum BountyCurrency {
+
+    /**
+     * Solar credits
+     *
+     */
+    CREDITS,
```

Comment:
Ik ariel wants this but in my opinion separating the two but allowing the use of both is a big mistake

Reply:
I didn't see a way to implement this better than an enum. It does sound like a cool feature, though.

Reply:
Why do you think It's a mistake ? Game wise or code wise?